packages/flutter/test/material/app_test.dart

```diff
@@ -143,4 +143,21 @@ void main() {
 
     expect(find.text('Home'), findsOneWidget);
   });
+
+  testWidgets('Could not implicitly use root initialRoute',
```

Comment:
I don't really understand the title of this test. Generally we want our titles to either describe the topic under test (e.g. "Default initial route") or the test itself (e.g. in this current case, "MaterialApp doesn't throw when omitting 'initialRoute' and only providing '/' route").